docs/data-sources/annotation.md

```diff
@@ -0,0 +1,82 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+subcategory: "Generic"
+layout: "aci"
+page_title: "ACI: annotation"
+sidebar_current: "docs-aci-data-source-annotation"
+description: |-
+  Data source for Annotation
+---
+
+# aci_annotation #
+
+Data source for Annotation
+
+## API Information ##
+
+* `Class` - [tagAnnotation](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/tagAnnotation/overview)
+
+* `Distinguished Name Formats`
+  - `TaskDeployCsr/annotationKey-[{key}]`
+  - `TaskDeployCtx/annotationKey-[{key}]`
+  - `TaskDeploySg/annotationKey-[{key}]`
+  - `TaskDeploySgRule/annotationKey-[{key}]`
+  - `TaskPolicyUpdate/annotationKey-[{key}]`
+  - `TaskResourceMap/annotationKey-[{key}]`
+  - `acct-[{name}]/acctoper/fault-{code}/annotationKey-[{key}]`
+  - `acct-[{name}]/adds/annotationKey-[{key}]`
+  - `acct-[{name}]/apigw/annotationKey-[{key}]`
+  - `acct-[{name}]/certstore-{store}/cert-[{name}]/certificateoper/fault-{code}/annotationKey-[{key}]`
+  - `acct-[{name}]/cloudsvc-[{name}]-typ-[{type}]-rg-{rg}/adds/annotationKey-[{key}]`
+  - `acct-[{name}]/cloudsvc-[{name}]-typ-[{type}]-rg-{rg}/apigw/annotationKey-[{key}]`
+  - `acct-[{name}]/cloudsvc-[{name}]-typ-[{type}]-rg-{rg}/cntreg/annotationKey-[{key}]`
+  - `acct-[{name}]/cloudsvc-[{name}]-typ-[{type}]-rg-{rg}/cosmosdb/annotationKey-[{key}]`
+  - `acct-[{name}]/cloudsvc-[{name}]-typ-[{type}]-rg-{rg}/dbricks/annotationKey-[{key}]`
+  - `acct-[{name}]/cloudsvc-[{name}]-typ-[{type}]-rg-{rg}/dbricks/rsbricksToPubSubnet/annotationKey-[{key}]`
+  - `acct-[{name}]/cloudsvc-[{name}]-typ-[{type}]-rg-{rg}/dbricks/rsbricksToPubSubnet/fault-{code}/annotationKey-[{key}]`
+  - `acct-[{name}]/cloudsvc-[{name}]-typ-[{type}]-rg-{rg}/dbricks/rsbricksToPvtSubnet/annotationKey-[{key}]`
+  - `acct-[{name}]/cloudsvc-[{name}]-typ-[{type}]-rg-{rg}/dbricks/rsbricksToPvtSubnet/fault-{code}/annotationKey-[{key}]`
+  - `acct-[{name}]/cloudsvc-[{name}]-typ-[{type}]-rg-{rg}/keyvault/annotationKey-[{key}]`
+  - `Too many DN formats to display, see model documentation for all possible parents.`
```

Comment:
Should we have DN formats for data_sources?

Reply:
Is there a reason to not display them? A data-source would point to an object in ACI and thus would also have a DN. This way you would be able to see the parent part of the DN. I would say then the information is consistent with the information displayed on resource side, so when somebody needs this info they will not need to switch to resource docs.